R/data.R

```diff
@@ -1,5 +1,52 @@
 #' WokeWindows citations
 #' @docType data
+#' @format
+#' An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 328141 rows and 45 columns.
+#' \describe{
+#'   \item{issuing_agency}{Boston Police division that issuing officer belongs to. a character vector.}
+#'   \item{agency_code}{Shorthand code for issuing agency. a character vector.}
+#'   \item{officer_id}{Unique identifier for BPD offficers. a character vector.}
+#'   \item{officer_name}{The name of the issuing officer.}
+#'   \item{event_date}{Year-Month-Day.}
+#'   \item{time_hh}{Hour of incident.}
+#'   \item{time_mm}{Minute of incident.}
+#'   \item{am_pm}{a character vector}
+#'   \item{viol_type}{The type of violator. Owner, operator, passenger, or bike.}
+#'   \item{citation_number}{"unique identifier for the citation; the same number will appear multiple times in the spreadsheet if the ticket is for multiple offenses." (Woke Windows)}
+#'   \item{citation_type}{Can be civil, criminal, or arrest.}
+#'   \item{offense}{"code for an offense; seems to correspond to a section in the Massachusetts General Laws e.g. '90160' is MGL c.90 §16" (Woke Windows).}
```

Comment:
Can we add a link to this? 

https://malegislature.gov/Laws/GeneralLaws/PartI/TitleXIV/Chapter90/Section16